xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampRequest.xml

```diff
@@ -158,15 +214,34 @@
         <Parameter Name="extensions" Type="System.Security.Cryptography.X509Certificates.X509ExtensionCollection" />
       </Parameters>
       <Docs>
-        <param name="signerInfo">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="requestedPolicyId">To be added.</param>
-        <param name="nonce">To be added.</param>
-        <param name="requestSignerCertificates">To be added.</param>
-        <param name="extensions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="signerInfo">The CMS signer information to build a timestamp request for.</param>
+        <param name="hashAlgorithm">The hash algorithm to use with this timestamp request.</param>
+        <param name="requestedPolicyId">The Object Identifier (OID) for a timestamp policy the Timestamp Authority (TSA) should use, or <see langword="null" /> to express no preference.</param>
+        <param name="nonce">An optional nonce (number used once) to uniquely identify this request to pair it with the response. The value is interpreted as an unsigned big-endian integer and may be normalized to the encoding format.</param>
+        <param name="requestSignerCertificates">
+          <see langword="true" /> to indicate the Timestamp Authority (TSA) must include the signing certificate in the issued timestamp token; otherwise, <see langword="false" />.</param>
+        <param name="extensions">An optional collection of extensions to include in the request.</param>
+        <summary>Create a timestamp request by hashing the signature of the provided signer with a specified algorithm.</summary>
+        <returns>An <see cref="T:System.Security.Cryptography.Pkcs.Rfc3161TimestampRequest" /> representing the chosen values.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+This constructor creates a timestamp for the signature value of a <xref:System.Security.Cryptography.Pkcs.SignerInfo> value,
+per the recommendation of IETF RFC 3161 Appendix A (Signature Time-stamp attribute using CMS).
```

Comment:
First time I see "Time-stamp" being used. Probably better to use the same word as everywhere else:
```suggestion
per the recommendation of IETF RFC 3161 Appendix A (Signature Timestamp attribute using CMS).
```

Reply:
Since I'm quoting the title of an appendix from another document, I think it best to use the style they use for that section header.